cactus/site.py

```diff
@@ -276,9 +275,12 @@ def build(self):
                 else:
                     os.remove(path)
 
+        render_start = datetime.now()
```

Comment:
consider using `datetime.utcnow()`. It is faster than now():

    In [5]: timeit('datetime.utcnow()', setup='from datetime import datetime')
    Out[5]: 0.6945295839977916

    In [6]: timeit('datetime.now()', setup='from datetime import datetime')
    Out[6]: 1.9823984030008432